src/shogun/converter/StochasticProximityEmbedding.cpp

```diff
@@ -29,19 +28,19 @@ void StochasticProximityEmbedding::init()
 {
 	SG_ADD(&m_k, "m_k", "Number of neighbors");
 	SG_ADD(&m_tolerance, "m_tolerance", "Regularization parameter");
-	SG_ADD(&m_max_iteration, "max_iteration", "maximum number of iterations");
+	SG_ADD(&m_num_updates, "m_num_updates", "SPE number of updates");
+	SG_ADD(&m_max_iteration, "max_iteration", "Maximum number of iterations");
```

Comment:
@gf712 are we still using SG_ADD; or is the templated watch already there?

Reply:
The PR is mostly ready I think. I can have a look tomorrow!